src/main/java/org/jabref/logic/msbib/MSBibMapping.java

```diff
@@ -22,7 +23,46 @@ public class MSBibMapping {
     private static final String BIBTEX_PREFIX = "BIBTEX_";
     private static final String MSBIB_PREFIX = "msbib-";
 
-    private static final HashBiMap<Field, String> BIBLATEX_TO_MS_BIB = HashBiMap.create();
+    private static final BiMap<Field, String> BIBLATEX_TO_MS_BIB = HashBiMap.create();
+
+    private static final BiMap<String, Integer> LANG_TO_LCID = HashBiMap.create();
+
+    static {
```

Comment:
Can you please add a comment with the link to the mapping so someone else knows where this comes from.
Otherwise it looks good to me now
https://learn.microsoft.com/en-us/openspecs/office_standards/ms-oe376/6c085406-a698-4e12-9d4d-c3b0ee3dbc4a